app/main.py

```
@@ -126,6 +135,13 @@ async def register(response: Response, register_data: Register):
             "INSERT INTO `chat_users`.`users` (user_id, password, active) VALUES (?, ?, ?);",
             [user_id, encrypted_password, 1]
         )
+        try:
+            sql_connection.cursor().execute(
+                "INSERT INTO `chat_users`.`statistics` (label, value) VALUES (?, 1) ON DUPLICATE KEY value = value +1;",
+                ['Registrations']
+            )
+        except mariadb.Error as error:
+            pass
```

Ruff (0.12.2)

Use `contextlib.suppress(mariadb.Error)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(mariadb.Error)`

(SIM105)